cluster-autoscaler/cloudprovider/gce/mig_info_provider_test.go

```diff
@@ -593,17 +663,93 @@ func TestRegenerateMigInstancesCache(t *testing.T) {
 				instancesToMig: map[GceRef]GceRef{},
 			},
 			fetchMigInstances: fetchMigInstancesFail,
+			projectId:         mig.GceRef().Project,
 			expectedErr:       errFetchMigInstances,
 		},
+		{
+			name: "bulkGceMigInstancesListingEnabled - fill empty cache for one mig - instances in creating/deleting state",
+			cache: &GceCache{
+				migs:                             map[GceRef]Mig{mig1.GceRef(): mig1},
+				instances:                        map[GceRef][]GceInstance{},
+				instancesToMig:                   map[GceRef]GceRef{},
+				migTargetSizeCache:               map[GceRef]int64{},
+				migBaseNameCache:                 map[GceRef]string{},
+				listManagedInstancesResultsCache: map[GceRef]string{},
+				instanceTemplateNameCache:        map[GceRef]InstanceTemplateName{},
+				migInstancesStateCache:           map[GceRef]map[cloudprovider.InstanceState]int64{},
+			},
+			fetchMigInstances:                 fetchMigInstancesConst(mig1Instances),
+			fetchMigs:                         fetchMigsConst([]*gce.InstanceGroupManager{mig1Igm}),
+			fetchAllInstances:                 fetchAllInstancesInZone(map[string][]GceInstance{"myzone1": {instance1, instance2}, "myzone2": {instance3}}),
```

Comment:
Can you leave a comment where the inconsistency comes from? I'm assuming instance3 should not be here?

Reply:
Done